### ISSUE TITLE
Notifications not working on Manjaro/KDE

### ISSUE THREAD

**Issue by BethAr** [sieren/QSyncthingTray#101]
Hey there,

I'm using QSyncthingTray on Manjaro with KDE and I'm getting no notifications, everything else seems to be working fine, with the exception of the other issue I posted. I don't know if this is a problem with QSyncthingTray or KDE - which can be rather flaky.

Thank you.

**Comment by sieren** [comment]
![screen shot 2016-04-02 at 13 24 41](https://cloud.githubusercontent.com/assets/5174440/14225988/5046e5a4-f8d6-11e5-9839-3411eb7412a1.png)

I had a look today and I was unable to reproduce this issue on Manjaro.

**Comment by BethAr** [comment]
The only notification I get is the one displayed on the screenshot, that is, when Syncthing is enabled or disabled. Should I also get notifications when the files are sync'ed? If not, I think this bug was a misapprehension on my part.

**Comment by sieren** [comment]
Ah okay. That's on purpose for now. Might add this later as soon as I have thought of a way to make them not too annoying (you probably dont want one each time a file gets synced)

**Comment by BethAr** [comment]
Ok, I was a bit baffled that it did fire a notification when it detected syncthing running, but nothing after that, so I concluded it was a bug.

SyncTrayzor, the client I'm running on Windows, seems to collect a number of changes and then notify, that way I get a "12 files were synchronized' notification, instead of one per file. I believe it also fires after a specific amount of time, because it does notify a single file change sometimes. The timing might be linked to how often I set the syncthing to check for files on each folder, but I have not checked.

Thanks for the clarification.
